src/main/java/org/alliancegenome/curation_api/dao/AGMDiseaseAnnotationDAO.java

```diff
@@ -16,7 +16,7 @@ protected AGMDiseaseAnnotationDAO() {
     }
 
     public List<String> findAllAnnotationCuries(String taxonID) {
```

Comment:
Maybe we should change the method name to findAllAnnotationUniqueIds or something similar?